tests/Behat/features/block-link-field.feature

```diff
@@ -0,0 +1,50 @@
+@javascript
+Feature: Add link to banner block
+  As a CMS user
+  I want to add link to banner blocks
+  So that my banners can have calls to actions
+
+  Background:
+    Given I add an extension "DNADesign\Elemental\Extensions\ElementalPageExtension" to the "Page" class
+      And a "image" "Uploads/folder1/file1.jpg"
+      And a "page" "Blocks Page" with a "Alice's Block" banner element with "Alice's sample content" content
+    Given I am logged in with "ADMIN" permissions
+      # Replace with 'And I click "Blocks Page" in the ".breadcrumbs-wrapper" element' once the ElementalArea refreshes,
+      # See https://github.com/dnadesign/silverstripe-elemental/issues/320
```

Comment:
Is this the wrong issue number?

Reply:
Not sure, I blindly copied the other feature test without reading the fine print. I can remove. this bit.

Reply:
Removed the comment.